api/config/cdr_versions_stable.json

```diff
@@ -21,7 +21,7 @@
     "creationTime": "2017-12-26 00:00:00Z",
     "releaseNumber": 1,
     "numParticipants": 946237,
-    "cdrDbName": "cdr20190215_test",
-    "publicDbName": "public20190215_test"
+    "cdrDbName": "synth_cdr_2019q1_1",
```

Comment:
Per Kayla's naming scheme, these should be:
synth_r_2019q1_1
synth_p_2019q1_1

https://docs.google.com/document/d/1W8DnEN7FnnPgGW6yrvGsdzLZhQrdOtTjvgdFUL6e4oc/edit

Reply:
Named it as this to prevent the permissions write issue. I
 was not sure if changing this to https://github.com/all-of-us/workbench/blob/master/api/db-cdr/generate-cdr/init-new-cdr-db.sh#L36 to ^p_ would need something to be re-run in order to write the permissions. 
Saw that this script is run as part of generate-cloudsql-db.